usr/share/rear/rescue/NETFS/default/610_save_capabilities.sh

```diff
@@ -1,5 +1,18 @@
-# save all found capapilities to file 
 
-if is_true "$NETFS_RESTORE_CAPABILITIES" ; then
-	getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities || Log "Error while saving capabilities to file."
-fi
+# Save all found capapilities.
+
+# Skip when the whole NETFS_RESTORE_CAPABILITIES array is empty.
+# For the 'test' one must have all array members as a single word i.e. "${name[*]}" because
+# the test should succeed when there is any non-empty array member, not necessarily the first one:
+test "${NETFS_RESTORE_CAPABILITIES[*]}" || return 0
+
+# Save capapilities to /var/lib/rear/recovery/capabilities:
+echo /dev/null > $VAR_DIR/recovery/capabilities
+
+# getcap and setcap are mandatory when NETFS_RESTORE_CAPABILITIES has a non-empty array member:
+has_binary getcap && has_binary setcap || Error "getcap and setcap are needed when NETFS_RESTORE_CAPABILITIES is non-empty"
+
+for directory in "${NETWORKING_PREPARATION_COMMANDS[@]}" ; do
```

Comment:
NETWORKING_PREPARATION_COMMANDS ?
Shouldn't it be NETFS_RESTORE_CAPABILITIES instead ?